.jenkinsci/artifacts.groovy

```diff
@@ -24,32 +22,21 @@ def uploadArtifacts(filePaths, uploadPath, artifactServers=['artifact.soramitsu.
       sh "gpg --yes --batch --no-tty --import ${CI_GPG_PRIVKEY} || true"
     }
     filePathsConverted.each {
-      sh "echo put ${it} $uploadPath >> \$(pwd)/batch.txt;"
+      sh "echo ${it} >> \$(pwd)/batch.txt;"
       sh "$shaSumBinary ${it} | cut -d' ' -f1 > \$(pwd)/\$(basename ${it}).sha256"
       sh "$md5SumBinary ${it} | cut -d' ' -f1 > \$(pwd)/\$(basename ${it}).md5"
       // TODO @bakhtin 30.05.18 IR-1384. Make gpg command options and paths compatible with Windows OS.
       if (!agentType.contains('MSYS_NT')) {
-        sh "echo \"${CI_GPG_MASTERKEY}\" | $gpgKeyBinary -o \$(pwd)/\$(basename ${it}).asc ${it}"
-        sh "echo put \$(pwd)/\$(basename ${it}).asc $uploadPath >> \$(pwd)/batch.txt;"
+        sh "echo \"${CI_GPG_MASTERKEY}\" | $gpgKeyBinary -o \$(pwd)/\$(basename ${it}).ascfile ${it}"
+        sh "echo \$(pwd)/\$(basename ${it}).ascfile >> \$(pwd)/batch.txt;"
       }
-      sh "echo put \$(pwd)/\$(basename ${it}).sha256 $uploadPath >> \$(pwd)/batch.txt;"
-      sh "echo put \$(pwd)/\$(basename ${it}).md5 $uploadPath >> \$(pwd)/batch.txt;"
+      sh "echo \$(pwd)/\$(basename ${it}).sha256 >> \$(pwd)/batch.txt;"
```

Comment:
Nexus calculates these hashsums for you after file uploaded. So no need to upload it yourself

Reply:
Yes, it does. But only in "browse" nexus mode. But in HTML View there are no hashes.
1. https://nexus.iroha.tech/#browse/browse:artifacts:libiroha%2Fbindings%2Fjava%2Fjava-bindings-Release-linux-20180902-fc5f88.zip
2. https://nexus.iroha.tech/service/rest/repository/browse/artifacts/libiroha/bindings/java/